WordPress/src/test/java/org/wordpress/android/ui/stats/refresh/lists/sections/granular/usecases/ReferrersUseCaseTest.kt

```diff
@@ -150,6 +155,7 @@ class ReferrersUseCaseTest : BaseUnitTest() {
             )
         ).thenReturn(contentDescription)
         whenever(statsUtils.toFormattedString(any<Int>(), any())).then { (it.arguments[0] as Int).toString() }
+        whenever(buildConfigWrapper.isJetpackApp).thenReturn(false)
```

Comment:
Shouldn't `buildConfigWrapper.isJetpackApp` return `true` because stats is only available to Jetpack? The same applies to all other tests where `buildConfigWrapper.isJetpackApp` is used.

Reply:
Yes. But the tests assume that being false.  Apparently the stats use cases have this if/then.